test/test.js

```diff
@@ -1,234 +1,232 @@
+      })
+    })
 
-    describe('function', function(){
+    describe('function', function () {
       var expected = {
         type: 'function',
         name: 'name',
         code: '\n  $some : "code";\n'
-      };
+      }
 
-      it('should detect it', function(){
-        var context = parser.contextParser(getContent('function.test.scss'));
-        assert.deepEqual(context, expected);
-      });
+      it('should detect it', function () {
+        var context = parser.contextParser(getContent('function.test.scss'))
+        assert.deepEqual(context, expected)
+      })
 
-      it('should detect it with linebreaks', function(){
-        var context = parser.contextParser(getContent('functionLinebreaks.test.scss'));
-        assert.deepEqual(context, expected);
-      });
-    });
+      it('should detect it with linebreaks', function () {
+        var context = parser.contextParser(getContent('functionLinebreaks.test.scss'))
+        assert.deepEqual(context, expected)
+      })
+    })
 
-    describe('variable', function(){
+    describe('variable', function () {
       var expected = {
         type: 'variable',
         name: 'name',
         value: '\'value\'',
         scope: 'private'
-      };
+      }
 
-      it('should detect it', function(){
-        var context = parser.contextParser(getContent('variable.test.scss'));
-        assert.deepEqual(context, expected);
-      });
+      it('should detect it', function () {
+        var context = parser.contextParser(getContent('variable.test.scss'))
+        assert.deepEqual(context, expected)
+      })
 
-      it('should detect it with linebreaks', function(){
-        var context = parser.contextParser(getContent('variableLinebreaks.test.scss'));
-        assert.deepEqual(context, expected);
-      });
+      it('should detect it with linebreaks', function () {
+        var context = parser.contextParser(getContent('variableLinebreaks.test.scss'))
+        assert.deepEqual(context, expected)
+      })
 
-      it('should detect it as global', function(){
-        var context = parser.contextParser(getContent('variableGlobal.test.scss'));
+      it('should detect it as global', function () {
+        var context = parser.contextParser(getContent('variableGlobal.test.scss'))
         assert.deepEqual(context, {
           type: 'variable',
           name: 'name',
           value: '\'value\'',
           scope: 'global'
-        });
-      });
+        })
+      })
 
-      it('should detect it with multiline Value', function(){
-        var context = parser.contextParser(getContent('variableMultilineValue.test.scss'));
+      it('should detect it with multiline Value', function () {
+        var context = parser.contextParser(getContent('variableMultilineValue.test.scss'))
         assert.deepEqual(context, {
           type: 'variable',
           name: 'map',
-          value: '(\n  \"a\": \"b\",\n  \"c\": \"\"\n)',
+          value: '(\n  "a": "b",\n  "c": ""\n)',
```

Comment:
There is no need to escape these characters (linter says). I think we used to use double quotes a while ago, hence the escape sequences.